config/reports/test-report.jinja2

```diff
@@ -1,16 +1,23 @@
 {# -*- mode: Python -*- -#}
 {# SPDX-License-Identifier: LGPL-2.1-or-later -#}
 
-{{ root.revision.tree }}/{{ root.revision.branch }} {{ '(Unknown)' if root.revision.describe is none else root.revision.describe}}: {{ total_runs }} runs, {{ total_failures}} failures
+{{ subject_str }}
 
-{{'%-15s %s %s'|format("test", "|", "result")}}
-{{'%s%s%s'|format("-"*16, "+", "-"*7)}}
-{%- for test in tests %}
-{{'%-15s %s %s'|format(test.name, "|", test.status)}}
-{%- endfor %}
+Tests Summary
+-------------
+
+Build details:
+
+Tree:     {{ root.revision.tree }}
+Branch:   {{ root.revision.branch }}
+Describe: {{ '(Unknown)' if root.revision.describe is none else root.revision.describe}}
+URL:      {{ root.revision.url }}
+SHA1:     {{ root.revision.commit }}
 
```

Comment:
@gctucker I am not totally sure about what should be in the template at this moment, because of the node model.